java/ql/src/experimental/Security/CWE/CWE-297/MissingSslEndpointIdentification.qhelp

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE qhelp PUBLIC
+  "-//Semmle//qhelp//EN"
+  "qhelp.dtd">
+<qhelp>
+
+<overview>
+<p>Java offers two mechanisms for SSL authentication - trust manager and hostname verifier. Trust manager validates the peer's certificate chain while hostname verification establishes that the hostname in the URL matches the hostname in the server's identification.</p>
```

Comment:
Use `<code>TrustManager</code>` and similarly HostnameVerifier, to make it clear we're talking about particular concrete interfaces here